cli/cmd/plugin/unmanaged-cluster/tanzu/tanzu.go

```diff
@@ -409,29 +412,56 @@ func (t *UnmanagedCluster) Delete(name string) error {
 	var err error
 	t.clusterDirectory, err = resolveClusterDir(name)
 	if err != nil {
+		log.Style(outputIndent, color.FgYellow).Warnf("Warning - could not resolve cluster config directory.\n")
+		log.Style(outputIndent, color.FgRed).Warnf("Cluster NOT deleted.\n")
```

Comment:
Reserve `color.FgRed` for errors. This could either be turned into an `Errorf` or `color.FgYellow` should be used.